plugins/package-managers/bazel/src/funTest/assets/projects/synthetic/bazel/.bazelversion

```diff
@@ -0,0 +1 @@
+7.0.1
```

Comment:
Once we upgrade Bazel, would then still Bazel version 7.0.1 be used on our CI ?
(If so, I suppose this would likely go unnoticed)

Reply:
Yes, but I think this is what we want, no ?
We need also tests that cover Bazel < 7.2.0